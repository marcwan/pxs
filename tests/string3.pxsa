        ADDVAR          y
        SUB             123, 456           , y
        DEBUGPRINT      y
MY_SUPER_AWESOME_LABEL:
        EXIT            0
