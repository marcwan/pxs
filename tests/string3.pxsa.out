DEBUGPRINT: y = -333.000000
